tests/mgr/test_cephmgr_validate_insights.py

```diff
@@ -54,6 +59,26 @@ def _ceph_run_insight(node, step_to_validate):
                 raise CephInsightOpsFailure(
                     f"Insights data doesn't have {section} section"
                 )
+    if step_to_validate == "verify-prune-health":
+        # Verify whether the prune-health command deletes the report generated
+        prune_timeout = 0  # Delete all logs
+        out = CephAdm(node).ceph.insights(prune=True, hours=prune_timeout)
+        sleep(prune_timeout)
```

Comment:
Constant sleep ? Try to check for some events to verify completion of previous command.